test/services/cache/ExportsItemBuilder.scala

```diff
@@ -103,9 +103,9 @@ trait ExportsItemBuilder {
     cache =>
       cache.copy(
         packageInformation = cache.packageInformation :+ PackageInformation(
-          typesOfPackages,
-          numberOfPackages,
-          shippingMarks
+          typesOfPackages.getOrElse(""),
```

Comment:
There would be nice to have same function signature here as PackageInformation constructor with some good defaults.